gulpfile.js

```diff
@@ -20,37 +20,49 @@ let sigS
 
 gulp.task('test:browser:before', (done) => {
   function createListenerA (cb) {
-    const id = PeerId.createFromJSON(
-        JSON.parse(
-          fs.readFileSync(
-            path.join(__dirname, './test/test-data/id-1.json'))))
-
-    const peerA = new PeerInfo(id)
-    const maA = multiaddr('/ip4/127.0.0.1/tcp/9100/ws')
-
-    peerA.multiaddr.add(maA)
-    swarmA = new Swarm(peerA)
-    swarmA.transport.add('ws', new WebSockets())
-    swarmA.transport.listen('ws', {}, echo, cb)
+    PeerId.createFromJSON(
+      JSON.parse(
+        fs.readFileSync(
+          path.join(__dirname, './test/test-data/id-1.json'))
+      ),
+      (err, id) => {
+        if (err) {
+          return cb(err)
+        }
```

Comment:
It would be super beneficial to have a libp2p module `libp2p-peer-gen` that already has 100 or 1000 PeerId generated and so, each time we request 'one peer', it would be super fast because it only had to load it from disk. We would use this everywhere for testing!